wandb/sdk/launch/__init__.py

```diff
@@ -57,7 +55,17 @@ def _run(
     Helper that delegates to the project-running method corresponding to the passed-in backend.
     Returns a ``SubmittedRun`` corresponding to the project run.
     """
-    project = fetch_and_validate_project(uri, experiment_name, api, runner_name, version, entry_point, parameters)
+    project = fetch_and_validate_project(
+        uri, experiment_name, api, runner_name, version, entry_point, parameters
+    )
+
+    if wandb_project is None:
+        wandb_project = api.settings("project")
```

Comment:
what does this default to? is there a project that would exist in the settings

Reply:
So long as there is a `wandb/settings` file the project and entity will be taken from there. Similarly the base_url, which will first be taken from ~/.config/wandb/settings then overwritten by the info in `wandb/settings`.

I'll add an "Uncategorized" option